Dynatrace/dynatrace-operator
# Add node collector service account to manifests

## Description

https://dt-rnd.atlassian.net/browse/K8S-11251

Add node collector service account to manifests

## How can this be tested?

make test/helm or make manifests and check if the dynatrace-node-config-collector service account is included
